x-pack/plugins/security_solution/public/common/components/visualization_actions/visualization_embeddable.tsx

```diff
@@ -8,7 +8,7 @@
 import React, { useCallback, useEffect, useRef } from 'react';
 import { useDispatch } from 'react-redux';
 import { css } from 'styled-components';
-import { ChartLabel } from '../../../overview/components/detection_response/alerts_by_status/chart_label';
+import { ChartLabel } from '../../../explore/components/alerts_by_status/chart_label';
```

Comment:
We should not import "sub-plugin" (neither _explore_ nor _overview_) stuff into _common_ components, maybe we should move all this stuff into `public/common/components`